login-workflow/src/components/StepperCard/StepperCard.tsx

```diff
@@ -1,19 +1,44 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import Card from '@mui/material/Card';
-import { useTheme } from '@mui/material/styles';
-import { Spinner } from './Spinner';
-import backgroundImage from '../assets/images/background.svg';
+import { SxProps, Theme, useTheme } from '@mui/material/styles';
+import { Spinner } from '../Spinner';
+import defaultBackgroundImage from '../../assets/images/background.svg';
 import { useInjectedUIContext } from '@brightlayer-ui/react-auth-shared';
 import Box from '@mui/material/Box';
 
+export type StepperCardProps = {
+    loading?: boolean;
+    backgroundImage?: string;
+    sx?: SxProps<Theme>;
+    cardStyles?: SxProps<Theme>;
+    loaderComponent?: ReactNode;
+};
+
 /**
- * Component that renders a blue textured background and centers its children in the screen.
+ * Component that renders a card centered on a blue textured background.
+ *
+ * @param loading if true a loading spinner will be displayed blocking the content
+ *
+ * @param backgroundImage will be displayed in the background instead of `defaultBackgroundImage`
  *
- * @param children the content to render on the blue background
+ * @param sx will override the styles for StepperCard root
+ *
+ * @param cardStyles will override the styles for `Card` component
+ *
+ * @param loaderComponent will be displayed instead of `Spinner`
  *
  * @category Component
  */
-export const BrandedBackground: React.FC<React.PropsWithChildren<React.PropsWithChildren>> = (props) => {
+export const StepperCard: React.FC<React.PropsWithChildren<React.PropsWithChildren<StepperCardProps>>> = (props) => {
```

Comment:
Remember to follow the naming convention. Instead of `cardProp`, it should be `CardProp`.

Reply:
> Remember to follow the naming convention. Instead of `cardProp`, it should be `CardProp`.

I think it makes sense to use `cardProps` as a prop name, like we do `primaryTypographyProps` in InfoListItem of react comp lib.

Reply:
- In [Info List Item's API doc](https://brightlayer-ui-components.github.io/react/components/info-list-item/api-docs), there is practically a prop that says `ListItemButtonProps`, instead of `listItemButtonProps`
- In [Drawer Nav Item's API doc](https://brightlayer-ui-components.github.io/react/components/drawer-nav-item/api-docs), there is also a prop called `InfoListItemProps`
- In [Drawer Rail Item's API doc](https://brightlayer-ui-components.github.io/react/components/drawer-rail-item/api-docs), there is a prop called `ButtonBaseProps`
- `primaryTypographyProps` might be a special case, because `primaryTypography` is not a component. I also did not see it anywhere in Info List Item's readme.

So I think `<ComponentNameInTitleCase>Props` is the standard naming convention for Material UI and Brightlayer UI.